modules/social_features/social_event/social_event.install

```diff
@@ -1621,3 +1621,20 @@ function social_event_update_8904() {
 function social_event_update_8905() {
   \Drupal::service('module_installer')->install(['social_event_addtocal']);
 }
+
+/**
+ * Show thumbnail on File fields when attachment is an image.
+ */
+function social_topic_update_8906() {
```

Comment:
Why is this inside social_event module?

Reply:
My bad, naming was incidentally copied incorrectly... 🤦